src/hotspot/share/gc/shenandoah/shenandoahMarkingContext.cpp

```diff
@@ -87,7 +87,8 @@ void ShenandoahMarkingContext::clear_bitmap(ShenandoahHeapRegion* r) {
 }
 
 bool ShenandoahMarkingContext::is_complete() {
-  return _is_complete.is_set();
+  bool result = _is_complete.is_set();
```

Comment:
Unnecessary change? Also seems unrelated to freeset changes.

Reply:
I'll fix that.  I had some debugging code in between those lines.